pkg/mcproducer/scripts/pull-image.sh

```diff
@@ -17,20 +19,20 @@ else
         exit 1
     fi
 
-    echo "File $kernel_module_image_filepath is not on the filesystem, pulling image "
-    podman pull --authfile /var/lib/kubelet/config.json $kernel_module_image
+    echo "File $kernel_module_image_filepath is not on the filesystem, pulling image $kernel_module_image:$kernel_module_image_tag"
```

Comment:
Should we introduce a variable for the full image name? We are concatenating `$kernel_module_image:$kernel_module_image_tag` several times.

Reply:
fixed